test/unit/units/kernel_load_test.py

```diff
@@ -64,32 +67,40 @@ def test_main_raises_on_kernel_load(
             call(
                 [
                     'kexec',
-                    '--load', '/system-root/boot/vmlinuz',
-                    '--initrd', '/system-root/boot/initrd',
+                    '--load', '/vmlinuz',
+                    '--initrd', '/initrd',
                     '--command-line', cmd_line
                 ]
             )
         ]
         assert mock_info.called
         assert mock_error.called
 
+    @patch('shutil.copy')
+    @patch('suse_migration_services.defaults.Defaults.get_system_root_path')
     @patch('suse_migration_services.logger.log.info')
     @patch('suse_migration_services.command.Command.run')
     @patch('suse_migration_services.units.kernel_load._get_cmdline')
     def test_main(
-        self, mock_get_cmdline, mock_Command_run, mock_info
+        self, mock_get_cmdline, mock_Command_run, mock_info,
+        mock_get_system_root_path, mock_shutil_copy
     ):
+        mock_get_system_root_path.return_value = '../data'
         cmd_line = \
             'root=UUID=ec7aaf92-30ea-4c07-991a-4700177ce1b8' + \
             'splash root=UUID=ec7aaf92-30ea-4c07-991a-4700177ce1b8 rw'
         mock_get_cmdline.return_value = cmd_line
         main()
+        assert mock_shutil_copy.call_args_list == [
+            call('../data/boot/vmlinuz', '/'),
+            call('../data/boot/initrd', '/')
```

Comment:
why do we need test/data/boot/... files to really exist ? I've seen the two empty files coming along with the pull request. I think that's not needed because the kexec call itself is also mocked